src/_locales/ja/messages.json

```diff
@@ -0,0 +1,723 @@
+            },
+            "link_start": {
+                "content": "$2",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "popup_instructions_no_trackers": {
+        "message": "$LINK_START$トラッカー$LINK_END$は検出されませんでした。",
+        "description": "Text shown in the popup when there are no trackers on the page.",
+        "placeholders": {
+            "link_start": {
+                "content": "$1",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "popup_blocked": {
+        "message": "このページには、サードパーティのリソースはありません。素晴らしい!",
+        "description": "Text shown in the popup when showing non-tracking domains is enabled, and there are no third-party domains on the page."
+    },
+    "intro_by_eff": {
+        "message": "EFF（電子フロンティア財団）のプロジェクト",
+        "description": ""
+    },
+    "tooltip_allow": {
+        "message": "スライダーを右に動かすと、ドメインが許可されます",
+        "description": ""
+    },
+    "intro_privacy_note": {
+        "message": "Privacy Badgerは、ユーザーが共有することを選択しない限り、ユーザーのブラウジングに関するデータを共有することは絶対にありません。",
+        "description": "Intro page paragraph."
+    },
+    "options_domain_list_trackers": {
+        "message": "Privacy Badgerはこれまでに、 $COUNT$個の潜在的な $TRACKER_LINK_START$トラッキングドメイン$TRACKER_LINK_END$ をブロックしました。",
+        "description": "Shown on the Tracking Domains tab on the options page after Privacy Badger learned to block one or more domains",
+        "placeholders": {
+            "count": {
+                "content": "$1",
+                "example": "900"
+            },
+            "tracker_link_start": {
+                "content": "$2",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "tracker_link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "options_domain_list_no_trackers": {
+        "message": "Privacy Badgerはまだ <a target='_blank' tabindex=-1 title='i18n_what_is_a_tracker' class='tooltip' href='https://www.eff.org/privacybadger#faq-What-is-a-third-party-tracker?'>トラッキングドメイン</a> を検出していません。引き続きブラウジングをお楽しみください！",
+        "description": "Shown on the Tracking Domains tab on the options page if all tracking domains have been removed."
+    },
+    "intro_not_an_adblocker_paragraph": {
+        "message": "Privacy Badgerは、3つの異なるウェブサイトで同じトラッカーが見つかるとブロックを開始します。スリーストライクでアウト!",
+        "description": "Intro page paragraph."
+    },
+    "options_domain_status_filter": {
+        "message": "ステータスで絞り込み:",
+        "description": "Label for a dropdown control on the Tracking Domains options page tab."
+    },
+    "report_text": {
+        "message": "そのエラーについて簡単に説明します。",
```

Comment:
Does this mean something like, "please describe the problem [you experienced on a website] in the text form located below this message"?

Reply:
I made a mistake!
I will correct that.